docs/release_notes/upgrading_to_destinations_v2.md

```diff
@@ -53,6 +53,17 @@ Whenever possible, we've taken this opportunity to use the best data type for st
 
 ![Upgrade Path](./assets/airbyte_destinations_v2_upgrade_prompt.png)
 
+:::caution Pay attention to the following before upgrade
```

Comment:
I think the `:::caution` needs to be on its own line maybe?

Reply:
nope, looks great!

<img width="508" alt="Screenshot 2024-02-09 at 6 09 33 PM" src="https://github.com/airbytehq/airbyte/assets/303226/d39d9ba7-7abd-4053-ae40-df64f8c7359f">